scripts/test_in_docker/test_in_docker.ps1

```diff
@@ -0,0 +1,28 @@
+﻿if ( -not (docker container ls | Select-String scylla_rust_driver)) {
+        echo "Scylla container must be running to execute tests (use 'make scylla-up')."
+        # exit 1
```

Comment:
It looks like it is commented out. Why?

Reply:
Accidentally left commented after tinkering with Windows. Will fix this.